eo-parser/src/main/resources/org/eolang/parser/add-probes.xsl

```diff
@@ -63,7 +63,8 @@ SOFTWARE.
         <xsl:apply-templates select="node()|@*"/>
         <xsl:for-each select="//o[starts-with(@base, '.')]">
           <xsl:variable name="p" select="eo:qualify(.)"/>
-          <xsl:if test="not(eo:contains-any-of($p, ('$', '^', '@','&lt;')))">
+          <xsl:variable name="c" select="string-length($p) - string-length(translate($p, '.', ''))"/>
```

Comment:
@mximp Why do you need to use `translate()` function here?
Do I understand correctly, that you wanna check that (pseudocode): 
```
length(Q.org.eolang) - length(Qorgeolang) > 1
```

Reply:
@volodya-lombrozo correct. I need to count the number of dots within FQN.

Reply:
Actually `>2` - it's a bug.. let me add one more commit

Reply:
I understand why you do it... It's a [common practice](https://stackoverflow.com/questions/36477331/count-number-of-character-in-a-variable-in-xslt-2-0). Interesting...